README.md

```diff
@@ -40,9 +40,9 @@ import { BrowserModule } from '@angular/platform-browser';
 import { TreeModule } from 'ng2-tree';
 
 @NgModule({
-  declarations: [MyComponent],
+  declarations: [AppComponent],
```

Comment:
Ah! I didn't realized that. I changed it cause the import above was for AppComponent. I will fix that.